docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -21,7 +21,7 @@ but it has distributed tracing implemented.
 You get out-of-the-box metrics from your tracing pipeline.
 
 {{< admonition type="note" >}}
-Metrics generation is disabled by default. Contact Grafana Support to enable metrics generation in your organization.
+Metrics generation is disabled by default. You can enable it for use with Application Observability defaults in Application Observability, or contact Grafana Support to enable metrics-generation for your organization with custom settings.
```

Comment:
Greetings. Thank you for taking the time to submit a change to the docs. What prompted the update to focus on Application Observability within this note specifically? 

This addition makes it sound like you can enable metrics-generator's span metrics processor *only* for use with App O11y instead of focusing on Tempo. 

If the goal is to let people know that enabling metrics-generator provides capabilities (or is required?) for App O11y in addition to the uses based around tracing data, then maybe we can phrase this differently. We can add text in another location that talks about where and how the metrics can be used, including supporting for Application Observability.

We can either include what features / capabilities are powered by metrics-generator in App O11y or we can link to the App O11y docs.

Reply:
Here's a suggestion: 

We could add a section or paragraph to the main [Metrics-generator page](https://grafana.com/docs/tempo/latest/metrics-generator/) that has something like : 

```
### Powered by metrics-generator

The metrics-generator processors also provide metrics to other Grafana products, including Application Observability.
Enabling the metrics-generator in your Grafana Cloud account enables default capabilities in Application Observability. 
```

The above is just an example text, not meant to be used. We'd need more specific content about how enabling in App O11y is/is not different from enabling metrics-generator for just Cloud account usage. Things like specific requirements or how to enable it for App O11y may be better suited in that documentation set instead of in the Tempo doc set.